docs/content/docs/connectors/table/formats/json.md

```diff
@@ -69,29 +69,33 @@ Format Options
       <tr>
         <th class="text-left" style="width: 25%">Option</th>
         <th class="text-center" style="width: 8%">Required</th>
+        <th class="text-center" style="width: 8%">Forwarded</th>
         <th class="text-center" style="width: 7%">Default</th>
         <th class="text-center" style="width: 10%">Type</th>
-        <th class="text-center" style="width: 50%">Description</th>
+        <th class="text-center" style="width: 42%">Description</th>
       </tr>
     </thead>
     <tbody>
     <tr>
       <td><h5>format</h5></td>
       <td>required</td>
+      <td>no</td>
       <td style="word-wrap: break-word;">(none)</td>
       <td>String</td>
       <td>Specify what format to use, here should be <code>'json'</code>.</td>
     </tr>
     <tr>
       <td><h5>json.fail-on-missing-field</h5></td>
       <td>optional</td>
+      <td>no</td>
```

Comment:
why is this no but `ignore-parse-errors` is?

Reply:
ignore parse errors seems safe? I don't have a strong opinion

Reply:
isn't the logic of `ignore-parse-errors` and `fail-on-missing-field` similar? I also don't have a strong opinion, but both should be consistent.